designer/server/src/main/scala/pl/touk/nussknacker/ui/notifications/Notification.scala

```diff
@@ -13,17 +15,23 @@ import pl.touk.nussknacker.ui.notifications.DataToRefresh.DataToRefresh
                                    toRefresh: List[DataToRefresh])
 
 object Notification {
-  def deploymentFailedNotification(id: String, name: ProcessName, failureMessageOpt: Option[String]): Notification = {
-    Notification(id, Some(name), s"Deployment of ${name.value} failed" + failureMessageOpt.map(" with reason: " + _).getOrElse(""),
+  def deploymentActionFailedNotification(id: String, actionType: ProcessActionType, name: ProcessName, failureMessageOpt: Option[String]): Notification = {
```

Comment:
In fact, these aren't only deployment actions. Hence, actionFaildNotification?

Reply:
Good point, on this level we can generalize it

Reply:
done